feat: add disableSanitizeConfirmation config option

duplabe opened the pull request.

duplabe committed:
feat: add disableSanitizeConfirmation config option

# Conflicts:
#	internal/config/k9s.go

# Conflicts:
#	internal/config/k9s.go
#	internal/config/k9s_int_test.go

duplabe commented:
@derailed 👋 Could you consider adding this config option to disable the sanitize confirmation?

derailed reviewed (changes requested):
@duplabe Thanks for this update! Tho I get the ack prompt could be bothersome, I don't this this option should remove the dialog completely. Perhaps a middle ground where the option just disable the required prompt? That way if the option is one and user accidentally hit `z` there is a safeguard. 
Also needs docs to explain this option does

derailed commented:
@duplabe Also looks like we have some conflicts. Can you take a peek?

duplabe committed:
refactor: show confirmation

duplabe commented:
@derailed I rebased the branch on your master, renamed the config option to `disableSanitizeConfirmationAck` and based on it's value I show the confirmation with, or without ack.
